src/components/ContentHighlights/data/hooks.js

```diff
@@ -67,8 +67,10 @@ export function useHighlightSet(highlightSetUUID) {
  */
 export function useContentHighlightsContext() {
   const setState = useContextSelector(ContentHighlightsContext, v => v[1]);
+  const getState = useContextSelector(ContentHighlightsContext, v => v[0]);
+  const getHighlightsState = useCallback(() => getState(), [getState]);
   // eslint-disable-next-line max-len
-  const currentSelectedRowState = useContextSelector(ContentHighlightsContext, v => v[0].stepperModal.currentSelectedRowIds);
+  const currentSelectedRowState = useContextSelector(ContentHighlightsContext, v => v[0].stepperModal?.currentSelectedRowIds);
```

Comment:
Why is the `?` necessary? Per the how the context value is generated in `ContentHighlightsContextProvider`, `stepperModal` _should_ always exist. Is there a test where its left off where it should be added versus making the actual code conditional?